CHANGES.rst

```diff
@@ -4,6 +4,8 @@ ChangeLog
 +------------+---------------------------------------------------------------------+------------+
 | Version    | Description                                                         | Date       |
 +============+=====================================================================+============+
+| *upcoming* | * Add ST7735 (160x80) Color LCD display driver                      |            |
```

Comment:
nit; maybe use *TBD* in date field

Reply:
and maybe mention the other modes as well? e.g. `[(160, 80), (160, 128), (128, 128)]`